lib/clients/AzureClient.py

```diff
@@ -41,6 +42,13 @@ def __init__(self, operation_name, configuration, directory_persistent, director
 
         self.snapshot_prefix = 'sf-snapshot'
         self.disk_prefix = 'sf-disk'
+        # scsi_host_number would be used to determine lun to device mapping
+        # scsi_host_number would be same for all data volumes/disks
+        self.scsi_host_number = self.get_hot_number_of_data_volumes()
```

Comment:
is there a typo ? `get_host_number_of_data_volumes` ?

Reply:
yes :) Fixing.